### DISCUSSION TITLE
Kibana does not show the data of elastiknn_sparse_bool_vector

### DISCUSSION THREAD

**Discussion by miqsoft** [alexklibisz/elastiknn#629] [Q&A]
Hey guys,

my kibana does not show the data in the elastiknn_sparse_bool_vector. In fact it tells that the field is empty.
However when accessing the data via an elasticsearch query the data is there.

Does one of you know whether this is the expected behaviour?
If it is not I am happy if one of you may have an idea why this occurs or how to fix it.

What I did so far:
- restart kibana and elasticsearch

Greetings and thanks in advance,
Michael

**Comment by alexklibisz** [comment]
Hi, I haven't tested with Kibana in a very long time. I suspect I might've broken some part of the internal API that's used by Kibana at some point. If you'd like to take a look, a good starting point would be these methods in the VectorMapper.scala file that are returning empty lists/arrays, e.g.:

* https://github.com/alexklibisz/elastiknn/blob/main/elastiknn-plugin/src/main/scala/com/klibisz/elastiknn/mapper/VectorMapper.scala#L82-L87
* https://github.com/alexklibisz/elastiknn/blob/main/elastiknn-plugin/src/main/scala/com/klibisz/elastiknn/mapper/VectorMapper.scala#L141

I recommend following the instructions for running w/ a debugger: https://github.com/alexklibisz/elastiknn/blob/main/developer-guide.md#debugging
, setting some breakpoints in these methods, and then use kibana to try to access the docs. See which methods get called and how they might be filled in.

**Comment by alexklibisz** [comment]
Converted this to an issue: #632
